src/graph/graphcontext_type.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Apache License, Version 2.0,
+ * modified with the Commons Clause restriction.
+ */
+
+#include "graphcontext.h"
+#include "graphcontext_type.h"
+#include "../index/index_type.h"
+
+/* Declaration of the type for redis registration. */
+RedisModuleType *GraphContextRedisModuleType;
+
+void GraphContextType_RdbSave(RedisModuleIO *rdb, void *value) {
```

Comment:
Add comment explaining what are we actually persisting and in which order, similar to the comments in bulk_insert.